packages/web3-eth-iban/package.json

```diff
@@ -39,7 +39,6 @@
 		"typescript": "^4.5.2"
 	},
 	"dependencies": {
-		"web3-common": "1.0.0-alpha.0",
```

Comment:
For Removing circular dependency https://github.com/ChainSafe/web3.js/pull/4742/commits/6b752ba74246d270a2b6e177b234e4ae69d9e5b2